frontend/src/components/ArtifactInfo.tsx

```diff
@@ -54,50 +52,35 @@ export class ArtifactInfo extends React.Component<ArtifactInfoProps, {}> {
 
   public render(): JSX.Element {
     const {artifact} = this.props;
-    const createTime = formatDateString(
-      getArtifactProperty(artifact, ArtifactProperties.CREATE_TIME) || '');
     return (
       <section>
-        <h1 className={commonCss.header}>{this.props.typeName} info</h1>
+        <h1 className={commonCss.header}>Type: {this.props.typeName}</h1>
+        <h2 className={commonCss.header2}>Properties</h2>
         <dl className={css.artifactInfo}>
-          <div className={css.field}>
-            <dt className={css.term}>Version ID</dt>
-            <dd className={css.value}>
-              {getArtifactProperty(artifact, ArtifactProperties.VERSION)}
-            </dd>
-          </div>
-          <div className={css.field}>
-            <dt className={css.term}>Workspace</dt>
-            <dd className={css.value}>
-              {getArtifactProperty(artifact, ArtifactCustomProperties.WORKSPACE,
-                true)}
-            </dd>
-          </div>
-          <div className={css.field}>
-            <dt className={css.term}>Run</dt>
-            <dd className={css.value}>
-              {getArtifactProperty(artifact, ArtifactCustomProperties.RUN,
-                true)}
-            </dd>
-          </div>
-          <div className={css.field}>
-            <dt className={css.term}>Description</dt>
-            <dd className={css.value}>
-              {getArtifactProperty(artifact, ArtifactProperties.DESCRIPTION)}
-            </dd>
-          </div>
-          <div className={css.field}>
-            <dt className={css.term}>Creation time</dt>
-            <dd className={css.value}>{createTime}</dd>
-          </div>
-          {this.props.typeName === 'Model' ?
-            <ModelInfo model={artifact} /> :
-            <div className={css.field}>
-              <dt className={css.term}>URI</dt>
-              <dd className={css.value}>{artifact.uri}</dd>
-            </div>
+          {Object.keys(artifact.properties || {})
+            // TODO: __ALL_META__ is something of a hack, is redundant, and can be ignored
+            .filter(k => k !== '__ALL_META__')
```

Comment:
*nit:* There's an enum value for `__ALL_META__` and a few of the other magic keys in the Api module if you want to use it. It seemed to have quite a few things packed under it in the first release but hopefully those become normal properties instead of being nested there.